coderd/pproflabel/pproflabel.go

```diff
@@ -0,0 +1,23 @@
+package pproflabel
+
+import (
+	"context"
+	"runtime/pprof"
+)
+
+// Go is just a convince wrapper to set off a labeled goroutine.
+func Go(ctx context.Context, labels pprof.LabelSet, f func(context.Context)) {
+	go pprof.Do(ctx, labels, f)
+}
+
+const (
+	ServiceTerraformProvisioner = "terraform-provisioner"
```

Comment:
Follow-up: would be cool to label pubsub as well.

Reply:
Which pubsub? The database pubsub listeners?

https://github.com/coder/coder/blob/cb1ec21f6155a566794766b151de4e39183d6822/coderd/database/pubsub/pubsub.go#L246-L246